# PickPlace.action

# Goal
geometry_msgs/PoseStamped object_pose
geometry_msgs/PoseStamped place_pose
int32 id

---
# Result
bool success
string message

---
# Feedback
string current_step

